darknet_ros_topic: /darknet_ros/bounding_boxes
output_bbx3d_topic: /darknet_ros_3d/bounding_boxes
point_cloud_topic: /realsense/depth/color/points
input_segment_topic: /ultralytics/segmentation/objects_segment

# this software requires that the X-axes points directly to the scene.
working_frame: map
mininum_detection_thereshold: 0.1
minimum_probability: 0.35
# interested_classes: ["orange","traffic light","chair","bottle","person", "italian_biscotti", "smoothie", "mango_juice", "crisps", "water", "sandwich", "toastie", "veggie_pot", "wrap", "espresso", "cappuccino", "americano","cup","table","bottle"]
interested_classes: ["chair", "person", "bottle", "table","bottle", "car", "truck"]
